### ISSUE TITLE
Create workshop metadata

### ISSUE THREAD

**Issue by zachlatta** [hackclub/hackclub#903]
Now that we're starting to build software around our workshops, like in [Lecture Hall](https://github.com/hackclub/lecture-hall), we need the ability to uniquely identify workshops.

I'd like to add a `metadata.yml` file to each workshop with the following info. The `id` field should be considered immutable and should never change.

```yml
id: {{random_8_character_string}}
name: {{replace}}
description: {{replace}}
```

Example for the Personal Website workshop:

```yml
id: 32ab8ede
name: Personal Website
description: Build your first website from scratch 💻
```

**Comment by zachlatta** [comment]
Think you could handle this, @ssun098?

**Comment by ssun098** [comment]
Sure! Although I don't understand where you get the id

**Comment by zachlatta** [comment]
Just go ahead and generate it using a string generator.

**Comment by ssun098** [comment]
PR at https://github.com/hackclub/hackclub/pull/904

**Comment by zachlatta** [comment]
Closed by #904.
